downstream/modules/platform/proc-operator-enable-https-redirect.adoc

```diff
@@ -0,0 +1,33 @@
+[id="proc-operator-enable-https-redirect"]
+
+= Enabling HTTPS redirect for single sign-on (SSO) for {Gateway} on {OCPShort}
+
+HTTPS redirect for SAML, allows you to log in once and access all of the {Gateway} without needing to reauthenticate.
+
+.Prerequisites 
+
+* You have successfully configured SAML in the gateway from the {OperatorPlatformNameShort}. Refer to link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/access_management_and_authentication/index#controller-set-up-SAML[Configuring SAML authentication] for help with this.
+
+.Procedure
+
+. Log in to {OCP}.
+. Go to menu:Operators[Installed Operators].
+. Select your {OperatorPlatformNameShort} deployment.
+. Select *All Instances* and go to your *AnsibleAutomationPlatform* instance.
+. Click the  {MoreActionsIcon} icon and then select btn:[Edit AnsibleAutomationPlatform].
+. In the *YAML view* paste the following YAML code under the `spec:` section:
+----
+spec:
+  extra_settings:
+    - setting: REDIRECT_IS_HTTPS
+      value: '"True"'
+
+----
```

Comment:
```suggestion
----
+
```

Reply:
Adding the "+" plus icon before and after the code block so it formats correctly and the next step is 7 rather than 1.